Jenkinsfile

```diff
@@ -263,23 +261,34 @@ pipeline {
         }
       }
     }
-    stage('Integration Tests') {
+    stage('Downstream Jobs') {
       agent none
       when {
         anyOf {
           changeRequest()
           expression { return !params.Run_As_Master_Branch }
         }
       }
+      environment {
+        GITHUB_CHECK_ITS_NAME = 'Integration Tests'
+        GITHUB_CHECK_OPBEANS_NAME = 'Opbeans'
+        ITS_PIPELINE = 'apm-integration-tests-selector-mbp/master'
+        OPBEANS_PIPELINE = 'apm-agent-java/opbeans-java-selector'
+      }
       steps {
-        log(level: 'INFO', text: 'Launching Async ITs')
         build(job: env.ITS_PIPELINE, propagate: false, wait: false,
               parameters: [string(name: 'AGENT_INTEGRATION_TEST', value: 'Java'),
                            string(name: 'BUILD_OPTS', value: "--java-agent-version ${env.GIT_BASE_COMMIT}"),
                            string(name: 'GITHUB_CHECK_NAME', value: env.GITHUB_CHECK_ITS_NAME),
                            string(name: 'GITHUB_CHECK_REPO', value: env.REPO),
                            string(name: 'GITHUB_CHECK_SHA1', value: env.GIT_BASE_COMMIT)])
         githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
+        build(job: env.OPBEANS_PIPELINE, propagate: false, wait: false,
+              parameters: [string(name: 'BUILD_OPTS', value: "JAVA_AGENT_REPO=${env.ORG_NAME}/${env.REPO_NAME} JAVA_AGENT_BRANCH=${env.GIT_BASE_COMMIT}"),
```

Comment:
What do you think? I initially thought to use two parameters:
- AGENT_REPO
- AGENT_VERSION

But I'm not fully sure if that's the same approach for all the opbeans so I decided to use a more generic approach with the BUILD_OPTS.